packages/create-aragon-app/package.json

```diff
@@ -78,7 +78,8 @@
     "eslint-plugin-standard": "^4.0.0",
     "node-fetch": "^2.3.0",
     "nyc": "^15.0.1",
-    "prettier": "^1.15.3"
+    "prettier": "^1.15.3",
```

Comment:
We can do this separately, but we might as well bump this up to latest too :)

Reply:
Good idea!